packages/react-reconciler/src/ReactFiberDevToolsHook.js

```diff
@@ -188,36 +187,25 @@ export function onCommitUnmount(fiber: Fiber) {
 }
 
 export function setIsStrictModeForDevtools(newIsStrictMode: boolean) {
-  if (consoleManagedByDevToolsDuringStrictMode) {
-    if (typeof log === 'function') {
-      // We're in a test because Scheduler.log only exists
-      // in SchedulerMock. To reduce the noise in strict mode tests,
-      // suppress warnings and disable scheduler yielding during the double render
-      unstable_setDisableYieldValue(newIsStrictMode);
-      setSuppressWarning(newIsStrictMode);
-    }
+  if (typeof log === 'function') {
+    // We're in a test because Scheduler.log only exists
+    // in SchedulerMock. To reduce the noise in strict mode tests,
+    // suppress warnings and disable scheduler yielding during the double render
+    unstable_setDisableYieldValue(newIsStrictMode);
+    setSuppressWarning(newIsStrictMode);
+  }
```

Comment:
We should get rid of this hack at some point. I don't think it was meant to survive the roll out.

Reply:
Yeah I'll fix that outside this diff now that I branched off it